README.md

```diff
@@ -21,29 +21,28 @@ If this behavior doesn't meet your requirements, the recommended way is to creat
 
 ### Configuration
 
-<a href="src/fluent-plugin-mdsd/out_mdsd_sample.conf">See configuration sample</a>
+[See configuration sample](./src/fluent-plugin-mdsd/out_mdsd_sample.conf)
 
 The mdsd output plugin is a buffered fluentd plugin. Besides supporting all the fluentd buffered plugin parameters, it supports the following required parameters
 
-  1) **djsonsocket**: this is the full path to mdsd dynamic json socket file.
+- **log_level**: this is [standard fluentd per plugin log level](http://docs.fluentd.org/v0.12/articles/logging#per-plugin-log). Valid values are: fatal, error, warn, info, debug, trace.
 
-  2) **acktimeoutms**: max time in milli-seconds to wait for mdsd acknowledge response. Before timeout, mdsd plugin will retry periodically to resend the events to mdsd. After timeout, the events holding in mdsd plugin memory will be dropped. If acktimeoutms is 0, the plugin won't do any failure retry if it cannot receives acknowledge from mdsd.
+- **djsonsocket**: this is the full path to mdsd dynamic json socket file.
 
-  3) **mdsd_tag_regex_patterns**: (Optional) An array of regex patterns for mdsd source name
-unification purpose. The passed will be matched against each regex, and if there's a match,
-the matched substring will be used as the resulting mdsd source name. For example, if the tag
-is `mdsd.ext_syslog.user.info` and the regex is `^mdsd\.ext_syslog\.\w+`, then `mdsd.ext_syslog.user`
-will be the mdsd source name.
-If this parameter is not specified, or for tags not matching any regexes in this array parameter,
-the original fluentd tag will be used as the mdsd source name. Default: `[]`.
+- **acktimeoutms**: max time in milli-seconds to wait for mdsd acknowledge response. Before timeout, mdsd plugin will retry periodically to resend the events to mdsd. After timeout, the events holding in mdsd plugin memory will be dropped. If acktimeoutms is 0, the plugin won't do any failure retry if it cannot receives acknowledge from mdsd.
```

Comment:
>milli-seconds [](start = 32, length = 13)

There is no hyphen in milliseconds; it's one word. Please make this change throughout the document.